services/.env.example

```diff
@@ -289,6 +289,5 @@ MOCK_COMMERCIAL_BANK_ETHIOPIA=TRUE
 # ---------------------------------------------------------------------
 
 # Playwright configuration
-BASE_URL=http://localhost:8888
```

Comment:
I think the ENV-variable should stay in this file, and maybe further explained with a comment above it (explaining/noting that its optional).

So:

```suggestion
# (Optional) URL of the interface/Portal to use in E2E-tests
BASE_URL=
```